linux_os/guide/system/accounts/accounts-restrictions/password_expiration/accounts_password_set_max_life_existing/tests/correct_stig.pass.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
```

Comment:
It is a coincidence that the default value of `var_accounts_maximum_age_login_defs` and the value selected by STIG profile are the same.

```suggestion
# variables = var_accounts_maximum_age_login_defs=60
```

Reply:
Added a comment to that effect with my other changes.

Reply:
I'm sorry,  I haven't found the comment you refer to, but I can see that the test scenario has stig in its name but it isn't limited to the stig profile and the variables aren't set to the values that are used in the stig profile.